app/helpers/phrase_usage_helper.rb

```diff
@@ -2,4 +2,30 @@ module PhraseUsageHelper
   def highlighted_survey_answer_html(survey_answer, phrase)
     survey_answer.gsub(/(#{phrase})/i){ |match| "<span class='phrase-highlight'>#{match}</span>" }.html_safe
   end
+
+  def phrase_usage_navigation_links
+    links = {}
+
+    if @presenter.pagination.page > 1
```

Comment:
The pattern I've almost always with helpers is to pass objects in, so to fit this style it should be `def phrase_usage_navigation_links(presenter)` and then use the presenter without the `@` in the method

Reply:
I can refactor this one out for all pages.